frontend/src/metabase/nav/components/ProfileLink/ProfileLink.jsx

```diff
@@ -47,6 +53,15 @@ function ProfileLink({ user, adminItems, onLogout }) {
       JSON.stringify(bugReportDetails),
     );
 
+    const helpLink = getHelpLink({
```

Comment:
What do you think about putting some of the hooks inside `getHelpLink` and converting it to a React hook? Because 3 of the props here:
- helpLinkSetting
- helpLinkCustomDestinationSetting
- isPaidPlan

Aren't used anywhere and they're just passed to `getHelpLink`

This looks a bit hard to read when a function receives so many arguments.

Reply:
Made it a hook, I also moved the test to the component as a result